src/nuget/Microsoft.DotNet.Build.Tasks.VersionTools.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.Build.Tasks.VersionTools</id>
+    <version>1.0.0-prerelease</version>
```

Comment:
All BuildTools package versions should stay in sync so we can branch reasonably.

Reply:
I followed the versioning of the other packages produced by BuildTools.  Are you proposing that all of these get updated?

```
Microsoft.DotNet.Build.CloudTest.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.Build.Tasks.Feed.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.Build.Tasks.VersionTools.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.2.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.ApiCompat.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.Cci.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.GenAPI.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.GenFacades.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.RepoUtils.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.Run.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.TestSuite.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.VstsBuildsApi.1.0.27-prerelease-02027-0.nupkg
Microsoft.DotNet.VersionTools.1.0.27-prerelease-02027-0.nupkg
Microsoft.xunit.extensibility.execution.netcore.1.0.1-prerelease-02027-0.nupkg
Microsoft.xunit.netcore.extensions.1.0.1-prerelease-02027-0.nupkg
xunit.console.netcore.1.0.4-prerelease-02027-0.nupkg
```